worker/src/main/java/gov/cms/ab2d/worker/processor/JobMeasure.java

```diff
@@ -15,38 +15,38 @@ public void update(ProgressTracker progressTracker, long value) {
     }, META_DATA_PROCESSED() {
         @Override
         public void update(ProgressTracker progressTracker, long value) {
-            progressTracker.addPatients((int) value);
+            progressTracker.addPatientsLoadedCount((int) value);
         }
-    }, BENE_REQUEST_QUEUED() {
+    }, PATIENT_REQUEST_QUEUED() {
         @Override
         public void update(ProgressTracker progressTracker, long value) {
             progressTracker.addPatientRequestQueuedCount((int) value);
         }
-    }, PATIENT_REQUEST_PROCESSED() {
+    }, PATIENT_REQUESTS_PROCESSED() {
         @Override
         public void update(ProgressTracker progressTracker, long value) {
             progressTracker.addPatientProcessedCount((int) value);
         }
+    }, PATIENT_REQUESTS_ERRORED() {
+        @Override
+        public void update(ProgressTracker progressTracker, long value) {
+            progressTracker.addPatientFailureCount((int) value);
+        }
     }, EOBS_FETCHED() {
         @Override
         public void update(ProgressTracker progressTracker, long value) {
-            progressTracker.addProcessedCount((int) value);
+            progressTracker.addEobFetchedCount((int) value);
         }
     }, EOBS_WRITTEN() {
         @Override
         public void update(ProgressTracker progressTracker, long value) {
-            progressTracker.addProcessedCount((int) value);
+            progressTracker.addEobProcessedCount((int) value);
         }
     }, EOBS_EMPTY() {
         @Override
         public void update(ProgressTracker progressTracker, long value) {
             throw new UnsupportedOperationException();
         }
-    }, EOBS_ERROR() {
```

Comment:
Not tracking this for the moment since our failure threshold is based on percentage of patients failing not eob related failures.